lib/data/serializers/product_info_serializer.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:memo/data/serializers/serializer.dart';
+import 'package:memo/domain/models/product_info.dart';
+
+class ProductInfoKeys {
+  static const id = 'productId';
```

Comment:
Caso resolva mudar para `id`, acho a melhor opção.

```suggestion
  static const id = 'id';
```